investment-app/TRINV.Api/TRINV.Domain/Entities/News.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace TRINV.Domain.Entities;
+
+using System.ComponentModel.DataAnnotations;
+using static Validations.EntityValidationConstants.News;
+
+public class News
+{
+    public News()
+    {
+        this.ViewsCount = 0;
+        this.UpVote = 0;
```

Comment:
Interger default value is 0. Remove the constructor

Reply:
Done.